Data Structures And Algorithms (DSA)/readme.md

```diff
@@ -0,0 +1,234 @@
+  </tr>
+  <tr>
+  <td><a href="https://youtu.be/58YbpRDc4yw?si=SCnvN0m2S_PFMy6K"> Linked List </a></td>
+  <td> As the above data structures, the linked list is also a linear data structure. But Linked List is different from Array in its configuration. It is not allocated to contiguous memory locations. Instead, each node of the linked list is allocated to some random memory space and the previous node maintains a pointer that points to this node. So no direct memory access of any node is possible and it is also dynamic i.e., the size of the linked list can be adjusted at any time. </td>
+  </tr>
+  <tr>
+  <td><a href="https://youtu.be/bxRVz8zklWM?si=yE2xewYRFknmumAY"> Stack </a></td>
+  <td> Stack is a linear data structure which follows a particular order in which the operations are performed. The order may be LIFO(Last In First Out) or FILO(First In Last Out). The reason why Stack is considered a complex data structure is that it uses other data structures for implementation, such as Arrays, Linked lists, etc. based on the characteristics and features of Stack data structure.</td>
+  </tr>
+  <tr>
+  <td><a href="https://youtu.be/W7uB9-TKfTg?si=zGRWmWUH11IxB3Zv"> Queue </a></td>
+  <td> A Queue is a linear structure which follows First In First Out (FIFO) approach in its individual operations. Another data structure that is similar to Stack, yet different in its characteristics </td>
+  </tr>
+  <tr>
+  <td><a href="https://youtube.com/playlist?list=PLkIpj7mL1E7uSElwIoJpgBnOQ8RJrYKzX&si=VAOW04OMQ_sXhhNL"> Trees </a></td>
+  <td> The first non-linear data structure you should learn is the tree. Tree data structure is similar to a tree we see in nature but it is upside down. It also has a root and leaves. The root is the first node of the tree and the leaves are the ones at the bottom-most level. The special characteristic of a tree is that there is only one path to go from any of its nodes to any other node.</td>
+  </tr>
+  <tr>
+  <td><a href="https://youtu.be/bvWVs0tJUOY?si=49dbm82Ck8_V2E70"> Graph </a></td>
+  <td> A Graph is a non-linear data structure consisting of a finite set of vertices(or nodes) and a set of edges that connect a pair of nodes. Another important non-linear data structure is the graph. It is similar to the Tree data structure, with the difference that there is no particular root or leaf node, and it can be traversed in any order. </td>
+  </tr>
+  <tr>
+  <td><a href="https://youtube.com/playlist?list=PLEJXowNB4kPwTb4BivkY0dENHmXdOEM3V&si=DiPfbRvuhacO00UA"> Searching Algorithms </a></td>
+  <td> Searching algorithms are used to find a specific element in an array, string, linked list, or some other data structure. </td>
+  </tr>
+  <tr>
+  <td><a href="https://youtube.com/playlist?list=PLEJXowNB4kPxse7iinEzb6FivjtQVxOXI&si=sfW3dRl3QQ8Veluq"> Sorting Algorithms </a></td>
+  <td> Sorting algorithms are used to sort a list of elements in a particular order. </td>
+  </tr>
```

Comment:
Are these resources really Data Structure or its Algorithm? Remove it or you can add a new section for tthe allgo